pythoncz/models/events.py

```diff
@@ -53,15 +54,31 @@ def load_events(ical_url):
     return calendar.events
 
 
+def find_first_url(text):
+    match = re.search(r'https?://[^ \n\r$]+', text or '')
```

Comment:
You can get tabs or whatnot. There is a symbol for whitespace, it's `\s`. However since we are doing reverse here, that's `\S`. Untested.

```suggestion
    match = re.search(r'https?://\S+', text or '')
```